client/src/components/Header.js

```diff
@@ -0,0 +1,14 @@
+import { Container, Row, Col, Image } from "react-bootstrap";
+import headerImg from "../img/header-img.png";
+
+export const Header = () => {
```

Comment:
In the screenshot this is attached to the menu, maybe changes need to be commited with the update as I cannot see its usage